pymeasure/instruments/lecroy/lecroyT3DSO1204.py

```diff
@@ -142,6 +148,40 @@ def _math_define_validator(value, values):
     return output
 
 
+class _ChunkResizer:
+    """The only purpose of this class is to resize the chunk size of the instrument adapter.
+    This is necessary when reading a big chunk of data from the oscilloscope like image dumps and
+    waveforms.
+    Note.
+    Only if the new chunk size is bigger than the current chunk size, it is resized. """
+
+    def __init__(self, adapter, chunk_size):
+        """ Just initialize the object attributes.
+        :param: adapter of the instrument. This is usually accessed through the
+        Instrument::adapter attribute.
+        :chunk_size: new chunk size (int).
+        """
```

Comment:
Do not add a docstring to the init, only to the class itself. The init will receive the class docstring as well. Therefore the class docstring should contain the information regarding the parameters.

Reply:
That's still true. Please move the parameters to the class docstring and remove the introduction sentence.

Reply:
see https://github.com/pymeasure/pymeasure/pull/697/files#r1033784985

Reply:
Done now.